process/block/baseProcess.go

```diff
@@ -2232,3 +2232,12 @@ func makeCommonHeaderHandlerHashMap(hdrMap map[string]data.HeaderHandler) map[st
 	}
 	return commonHdrMap
 }
+
+func waitForHeaderHashes(waitTime time.Duration, chanRcvHeaderHashes chan bool) error {
+	select {
+	case <-chanRcvHeaderHashes:
+		return nil
+	case <-time.After(waitTime):
```

Comment:
please use time.Timer instead of time.After

Reply:
Refactored to use `timer := time.NewTimer(waitTime)`